.circleci/config.yml

```
@@ -810,6 +825,11 @@ jobs:
     parallelism: 6
     <<: *steps_soltest_all
 
+  t_ubu_lsp: &t_ubu_lsp
+    <<: *base_ubuntu2004
```

Are tests multithreaded or use a significant amount of RAM? If not I'd use a `small` instance here:
```suggestion
    <<: *base_ubuntu2004
    resource_class: small
```

Many of our jobs [could actually run on `small`](https://github.com/ethereum/solidity/issues/11670#issuecomment-954297346) and I'm changing that in #12215 but I can't change a job that does not exist yet, so you need to change it here :)